bin/common.bash

```diff
@@ -136,18 +136,23 @@ validate_config() {
 
         file="${1}"
         options=$(yq r -p p "${2}" '**')
-        # Loop all lines in $2 and throws error if same option is not
+        # Loop all lines in $2 and warns if same option is not
         # available in $1
-        will_exit="false"
+        maybe_exit="false"
         for opt in ${options}; do
             value=$(yq r "${file}" "${opt}")
             if [[ -z "$value" ]]; then
-                log_error "ERROR: ${opt} is not set in ${file}"
-                will_exit="true"
+                log_warning "WARN: ${opt} is not set in ${file}"
+                maybe_exit="true"
             fi
         done
-        if [[ "${will_exit}" == "true" ]]; then
-            exit 1
+
+        if [[ "${maybe_exit}" == "true" ]]; then
+            echo -n -e "[\e[34mck8s\e[0m] Do you want to exit? (y/n): " 1>&2
```

Comment:
If we're going to keep the input requirement, maybe use the word "abort" or rephrase to "Continue anyway?" instead of "exit"?